repo/db/profile.go

```diff
@@ -15,32 +15,6 @@ func NewProfileStore(db *sql.DB, lock *sync.Mutex) repo.ProfileStore {
 	return &ProfileDB{db, lock}
 }
 
-func (c *ProfileDB) Init(id string, secret []byte) error {
```

Comment:
no longer needed because the master key is just the IPFS private key, which we can get direct from the ipfs config